src/admin/FAQAdmin.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace SilverStripe\Faq\Admin;
+
+use SilverStripe\Admin\ModelAdmin;
+use SilverStripe\Faq\Model\FAQ;
+
+/**
+ * Model Admin for FAQs search module.
+ * Allows a content author to publish and edit questions and answers.
+ *
+ * @see FAQ for FAQ DataObject.
+ */
+class FAQAdmin extends ModelAdmin
+{
+    /**
+     * @var string
+     */
+    private static $url_segment = 'faq';
+
+    /**
+     * @var array
+     */
+    private static $managed_models = [
+        FAQ::class,
+    ];
+
+    /**
+     * @var string
+     */
+    private static $menu_title = 'FAQs';
+
+    /**
+     * @var array
+     */
+    private static $model_importers = [
+        'FAQ' => FAQCsvBulkLoader::class,
```

Comment:
Missing namespace :)

Reply:
you mean namespace for the `FAQCsvBulkLoader` or `FAQ`?  
`FAQCsvBulkLoader` and `FAQAdmin` are in the same namespace so you don't need to add the namespace for  `FAQCsvBulkLoader`, 
For `FAQ` its just a name no need namespace.

Reply:
For `FAQCsvBulkLoader`, right I thought it was also used for clarity and readability purposes